org.eclipse.jdt.debug.ui/plugin.xml

```diff
@@ -3865,6 +3865,21 @@ M4 = Platform-specific fourth key
             type="org.eclipse.swt.widgets.Composite"
             description="%descriptionSWTComposite"/>
  </extension>
+ 	<extension
+	point="org.eclipse.ui.popupMenus">
+	<viewerContribution
+		targetID="org.eclipse.debug.ui.BreakpointView"
+		id="org.eclipse.jdt.debug.ui.removeAllConditions">
+		 <action
+               label="%DisableAllConditions.label"
+               helpContextId="remove_all_breakpoints_action_context"
+               icon="$nl$/icons/full/elcl16/removeAllConditions.png"
+               class="org.eclipse.jdt.internal.debug.ui.actions.RemoveAllCondtionalBreakpoints"
+               menubarPath="breakpointGroupJava"
```

Comment:
For that there is a small issue.
the current menu items ("Enable" - "Paste") are from platform and their ordering is based on the code arrangement of the action sets
<img width="332" alt="image" src="https://github.com/user-attachments/assets/241158c2-9dea-489d-90c1-66dac97abb46">
If I provide the same group name of those, then the new plugin will stack on top of that, and I couldnt find any postion or ordering attribute to position it, so can we go with existing ?
<img width="312" alt="image" src="https://github.com/user-attachments/assets/64082c77-3e6b-4aed-b33f-26b1c5a39c03">

Reply:
Hi @iloveeclipse can we go with bottom placement ?
As it is a new grouping we can include more breakpoints options specific to JDT only
eg Hit points bp, exception class bp, watchpoints etc..

Reply:
The right solution would be to add additional separator or group at the platform and contribute new menu to that, so it is not placed in a wrong location. Please try that.

Reply:
Will create a separate PR in platform for which includes additional separator.

Reply:
By making additional changes in platform it will be fixed.

<img width="343" alt="image" src="https://github.com/user-attachments/assets/f2eeee8e-8aec-4e9a-8417-2136fd3f8a14">

Reply:
<img width="470" alt="image" src="https://github.com/user-attachments/assets/97142c12-9dc5-4422-8af5-87a5502c614e">

Reply:
For the platform changes ,raised -> https://github.com/eclipse-platform/eclipse.platform/pull/1648